### PR TITLE
removing travis CI, being replaced by github actions

### LINKED ISSUE #599: Disable travis CI as the testing infrastructure works now on github actions

### PR DESCRIPTION
**Pull request by grzanka**
Fixes #599

### CODE REVIEW

File: docs/technical.rst [hunk 1]
```diff
@@ -22,18 +20,14 @@ Technical documentation
     * - docs
       - |docs|
     * - tests
-      - |travis| |appveyor|
+      - |appveyor|
```
Comment (michal367) [inline docs/technical.rst:23]: missing github actions

Comment (grzanka) [reply docs/technical.rst]: done in d30d937

File: docs/technical.rst [hunk 2]
```diff
@@ -22,22 +20,22 @@ Technical documentation
...
 
-.. |travis| image:: https://travis-ci.org/pytrip/pytrip.svg?branch=master
-    :alt: Travis-CI Build Status
-    :target: https://travis-ci.org/pytrip/pytrip
-
 .. |appveyor| image:: https://ci.appveyor.com/api/projects/status/github/pytrip/pytrip?branch=master&svg=true
     :alt: Appveyor Build Status
     :target: https://ci.appveyor.com/project/grzanka/pytrip
 
+.. |ghactions| image:: https://github.com/github/docs/actions/workflows/main.yml/badge.svg?branch=master
```
Comment (michal367) [inline docs/technical.rst:35]: Bad url. Pytrip doesn't have `main.yml` workflow

Comment (grzanka) [reply docs/technical.rst:35]: Thanks, I will fix

Comment (grzanka) [reply docs/technical.rst:35]: image

**Review by michal367 — APPROVED** [review]
Ok

**Pull request merged**